docs/reference/release-notes/870.md

```diff
@@ -29,6 +29,10 @@ These release notes identify the new features included in 8.7, including [alpha
 
 ### Amazon OpenSearch Optimize support <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span><span class="badge badge--medium" title="This feature affects Optimize">Optimize</span>
 
+:::note
+This feature was originally released with 8.7.0-alpha3, and is no longer available in 8.7.0. Amazon OpenSearch Optimize support is now available in [8.8.0-alpha1](./880.md#amazon-opensearch-optimize-support-self-managedoptimize). For more information, see the Camunda 8.7 and 8.8 [release update blog](https://camunda.com/blog/2025/01/camunda-87-88-release-update/).
```

Comment:
I'm having a battle with tenses in my head here, but I like that the note now covers all the objectives/goals.